source/common/http/hash_policy.cc

```diff
@@ -33,21 +36,33 @@ class HeaderHashMethod : public HashMethodImplBase {
     }
   }
 
+  const std::string HEADER_VALUE_SEPARATOR = ",";
```

Comment:
no static non-pods. You can use `constexpr absl::string_view` here.

See https://github.com/envoyproxy/envoy/blob/main/STYLE.md
> The Google C++ style guide points out that non-PoD static and global variables are forbidden. This includes types such as std::string. We encourage the use of the advice in the C++ FAQ on the static initialization fiasco for how to best handle this.

This is a general note, but in fact I think you don't need this (see below).

Reply:
thanks, good to know that anyway